src/display.js

```diff
@@ -0,0 +1,64 @@
+import LS from './localstorage.js';
+
+function Display() {}
+const ls = new LS();
+
+Display.prototype.showAllTodos = function () {
+  const items = ls.fetchItem();
+  let newHtml = '';
+  items.forEach((itm) => {
+    newHtml += `
+    <div class="task ${itm.isComplete ? 'completed' : ''}" data-createdate="${itm.id}">
+    <div class="taskDetails">
+          <input type="checkbox" class="task-check" ${itm.isComplete ? 'checked' : ''} />
+          <label class="taskTitle">${itm.title}</label>
+    </div>
+
+    <div class="taskIcon">
+          <ion-icon class="taskIconEdit" name="create-outline"></ion-icon> 
+          <ion-icon class="taskIconDelete" name="trash-outline"></ion-icon>
+    </div> 
+  </div>`;
+  });
+  document.querySelector('.taskList').innerHTML = newHtml;
+};
+
+Display.prototype.addToDisplay = function (item) {
+  ls.storeTodo(item);
+
+  const newHTML = `
+    <div class="task" data-createdate="${item.id}">
+        <div class="taskDetails">
+              <input type="checkbox" class="task-check" />
+              <label class="taskTitle">${item.title}</label>
+        </div>
+
+        <div class="taskIcon">
+              <ion-icon class="taskIconEdit" name="create-outline"></ion-icon>
+              <ion-icon class="taskIconDelete" name="trash-outline"></ion-icon>
+        </div>
+    </div>
+    `;
+
+  document.querySelector('.taskList').insertAdjacentHTML('afterbegin', newHTML);
+};
+
+Display.prototype.resetForm = function () {
+  document.querySelector('#newTaskID').value = '';
+};
+
+Display.prototype.deleteTodo = function (e) {
+  const item = e.target.parentElement.parentElement;
+  const id = item.dataset.createdate;
+  ls.deleteTodo(id);
+  item.remove();
+};
+
+Display.prototype.completeTodo = function (e) {
+  const item = e.target.parentElement.parentElement;
+  const id = item.dataset.createdate;
+  ls.completeTodo(id);
+  item.classList.toggle('completed');
+};
```

Comment:
- [ ] As I mentioned in my above comment, your ID should be number based not `Date and Time` based, no matter if the user is adding a new task or deleting the task, Your ID should be assigned to all the todos uniquely but should be number based. Let's take an example: 
  - If there are already two todos added by the user, then if the user adds the third one then the ID should be 3 Or if the user deletes any of the todos lets say out of 4 then the last todo id should be 3.